meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/kernel-module-cldatdrv/kernel-module-cldatadrv_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "CPURTT drive Module"
+LICENSE = "CLOSED"
+
+inherit module
+
+PR = "r0"
+PV = "0.1"
+
+SRC_URI = "git://github.com/iusyk/cluster_data_drv.git;ptotocol=http;branch=master"
```

Comment:
Maybe it's better to use HTTPS protocol instead of HTTP.

Reply:
Please move repo to `xen-troops` from `iusyk`.

Reply:
the recipe has been removed, the module is integrated in the linux